### PR TITLE
Animate info panel.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review. It adds an animation for sliding on the info panel in a conversation. I'm not sure how to add an animation for sliding it off.

### CODE REVIEW

**Comment by adamjspooner** [comment]
@keybase/react-hackers, bump.

**Comment by chrisnojima** [comment]
lets not check in flow-typed files that are entire stubs. it stubs out a ton of files we never need so lets just remove it.
you can also remove this line from package.json
```
    "build-flow-stubs": "flow-typed create-stub",
```
just add the line to flow-interface.js.flow instead

**Comment by adamjspooner** [comment]
@keybase/react-hackers, this is ready for another look.

**Comment by adamjspooner** [comment]
@chrisnojima, how does this look?

File: shared/common-adapters/animated.js.flow [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// @flow
+import * as React from 'react'
+export type Props = {|
+  children?: Function,
```
Comment (chrisnojima) [inline shared/common-adapters/animated.js.flow]: should be something tighter like
```
children: (styles: CrossPlatformStyles) => React.Node
```

File: shared/chat/conversation/info-panel/container.js [hunk 2]
```diff
@@ -156,24 +157,33 @@ class InfoPanelSelector extends React.PureComponent<Props> {
     return isMobile ? (
       <ConnectedInfoPanel onBack={this.props.onBack} conversationIDKey={this.props.conversationIDKey} />
     ) : (
-      <Box onClick={this.props.onBack} style={clickCatcherStyle}>
-        <Box style={panelContainerStyle} onClick={evt => evt.stopPropagation()}>
-          <ConnectedInfoPanel onBack={this.props.onBack} conversationIDKey={this.props.conversationIDKey} />
-        </Box>
-      </Box>
+      <Kb.Box onClick={this.props.onBack} style={clickCatcherStyle}>
```
Comment (chrisnojima) [inline shared/chat/conversation/info-panel/container.js]: this only shows desktop, we should have some example of mobile working also

Comment (adamjspooner) [reply shared/chat/conversation/info-panel/container.js]: I wasn't sure what trivial thing to animate on mobile.

File: shared/common-adapters/animated.js.flow (hunk 1 above)
Comment (adamjspooner) [reply shared/common-adapters/animated.js.flow]: It's different between desktop and native.

**Pull request closed without merging**